person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    true if of_age? || @parent_permission
+  end
```

Comment:
- [ ] There seems to be unexpected behavior with your function here. Your `can_use_services?` method returns **`nil`** when I set up the instance in the image below. But it should return **`false`** instead since **age** is less than 18 and **permission** is false. This because the method returns a true condition otherwise the method returns default value of `nil`. Kindly fix this by removing `true if` so that you simply have only `of_age? || @parent_permission`.

  ![nicu-can-use](https://user-images.githubusercontent.com/36542986/150886956-06d56532-724c-4a89-aa02-46d2e1fd21a1.jpg)

Reply:
fixed, thanks a lot for the feedback!